### PR TITLE
Remove unused functions

### PR DESCRIPTION
**Pull request by charleskawczynski**
Remove unused functions, fix some typos. I don't think these extra functions are needed, but it would be nice to get confirmation.

### CODE REVIEW

**Review by trontrytel — APPROVED** [review]
LGTM. Not sure about the `entr_detr_env_moisture_deficit_b_ED_MF` not being needed at some point @yairchn ?

File: src/turbulence_functions.jl [hunk 1]
```diff
@@ -1,50 +1,5 @@
 # Entrainment Rates
 
-function entr_detr_env_moisture_deficit_b_ED_MF(entr_in::entr_struct)
```
Comment (yairchn) [inline src/turbulence_functions.jl:3]: I would like to keep this version of the entrainment closure - we might find ourselves using it

**Review by yairchn — COMMENTED** [review]
please keep the entrainment functions that are there

**Pull request merged**